indico/modules/events/editing/operations.py

```diff
@@ -269,38 +302,44 @@ def create_new_tag(event, code, title, color, system=False):
     return tag
 
 
-def update_tag(tag, code=None, title=None, color=None):
-    if code:
-        tag.code = code
-    if title:
-        tag.title = title
-    if color:
-        tag.color = color
+def update_tag(tag, updates):
+    changes = tag.populate_from_dict({k: v for k, v in updates.items() if v is not None})
     db.session.flush()
     logger.info('Tag %r updated by %r', tag, session.user)
+    log_fields = {'code': 'Code', 'title': 'Title', 'color': 'Color'}
+    tag.event.log(EventLogRealm.management, LogKind.change, 'Editing', f'Tag {tag.verbose_title} updated',
+                  session.user, data={'Changes': make_diff_log(changes, log_fields)})
```

Comment:
By adding `meta={'editing_tag_id': tag.id}` to all the tag-related log calls (or by creating a `log()` method in `EditingTag` similar to the ones we have in `Contribution` etc.) one could easily use the filter button in the log viewer to see all operations related to that tag. Not super important for tags but maybe nice to use nonetheless whenever you are operating on a specific object that has an ID anyway.